### PR TITLE
Full adder

### PR DESCRIPTION
**Pull request by Nesathurai**
Added an example circuit for the full adder

### CODE REVIEW

File: site/public/examples/examples.json [hunk 1]
```diff
@@ -9,6 +9,11 @@
             "name": "SR-Latch Gate Implementation",
             "file": "example2.circuit",
             "thumbnail": "./img/thumbnails/example2.png"
-        }
+        },
+	{
+            "name": "Full Adder Circuit",
+            "file": "Full Adder Circuit.circuit",
+            "thumbnail": "./img/thumbnails/example2.png"
+	}
```
Comment (LeonMontealegre) [inline site/public/examples/examples.json:17]: Braces look misaligned here, make sure to be using 4-spaces and not tabs because tabs can be evil

Comment (LeonMontealegre) [reply site/public/examples/examples.json:17]: By here, I mean the braces are misaligned with the rest of the braces in the file

Comment (Nesathurai) [reply site/public/examples/examples.json:17]: Ok will do. Am I using Github correctly? I havent really used it much before.

Comment (LeonMontealegre) [reply site/public/examples/examples.json:17]: Yeah! All you gotta do now is make the changes and then add/commit the files and push again and it'll automatically update this pull request

**Comment by Robert-Hammond** [comment]
Looks good!

**Pull request merged**